pkg/version/version.go

```diff
@@ -33,6 +35,24 @@ var (
 	ReleaseStatus = "unreleased"
 )
 
+func GetMajorVersion() string {
+	v, err := semver.Parse(Version[1:])
+	if err != nil {
+		println(err)
+		return "0"
+	}
+	return fmt.Sprint(v.Major)
+}
+
+func GetMinorVersion() string {
+	v, err := semver.Parse(Version[1:])
+	if err != nil {
+		println(err)
+		return "0"
+	}
+	return fmt.Sprintf("%d.%d", v.Minor, v.Patch)
```

Comment:
Why merge minor and patch to minor? I think k8sversion.Info's intention is major for major, minor for minor, gitversion for full version (the `Version` in this file):
```
version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-13T11:13:49Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

Reply:
I didn't know the git version is used as full version, I would update it.

Reply:
Updated.